### PR TITLE
Adding persist to make it use dask

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

File: examples/dashboard.yml [hunk 1]
```diff
@@ -36,11 +37,11 @@ sources:
           label: Drop-off Hour
         pickup_hour:
           label: Pick-up Hour
-      plot:
```
Comment (philippjfr) [inline examples/dashboard.yml]: I'm confused why was this removed?

Comment (jsignell) [reply examples/dashboard.yml]: I was using the github UI. Just a mistake

**Comment by jsignell** [comment]
I'm going to merge this since it's passing. Sound good @jbednar?
